src/components/Form/SelectDropdown.tsx

```diff
@@ -80,6 +101,7 @@ const MultiSelectDropdown = ({
         // @ts-ignore
         onChange={handleChange}
         className="dark:text-white"
+        styles={style}
```

Comment:
There is actually something in the docs about using classes:
https://react-select.com/styles#using-classnames

It might be cleaner to use these classes alongside [@apply](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply) to utilize tailwindcss classes.